assets/javascripts/discourse/connectors/admin-menu/nav-item.hbs

```diff
@@ -1,4 +1,4 @@
 {{#if currentUser.admin}}
-  {{nav-item route='adminTicketing' label='ticketing.title'}}
+  {{nav-item route='adminTicketing' label='discourse_ticketing.title'}}
```

Comment:
Can we try `discourse_ticketing.dashboard.title` to avoid potential namespace collisions in the future?